package.json

```diff
@@ -28,6 +28,7 @@
     "bootstrap-vue": "^2.0.0-rc.11",
     "bpmn-moddle": "^5.1.5",
     "bpmnlint": "^5.0.0",
+    "bpmnlint-plugin-processmaker": "file:processmaker-plugin",
```

Comment:
Where is this package being hosted?

Reply:
Locally; The code is included in this PR, it refers to the `processmaker-plugin` folder.

Reply:
This is how they have it set up in their example:
https://github.com/bpmn-io/bpmnlint-playground/blob/f35cc2df5ad74ddde4ee8e7cef319dbc2af1bf77/package.json#L22